Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN yarn install --ignore-scripts --non-interactive --frozen-lockfile && \
 FROM ${RUNTIME_IMAGE}
 WORKDIR /spoke
 COPY --from=builder /spoke/build build
+COPY --from=builder /spoke/migrations migrations
```

Comment:
Why is this needed?

Reply:
It is not! I was just confirming, but because of how I have this sequenced in my docker-compose file, the postgres database is creating while Spoke is spinning up, which I was having trouble with because even with `SUPPRESS_DATABASE_AUTOCREATE` the migrations were still being called.

I just read 10 lines-ish below and saw that I can just have the same effect by setting `SUPPRESS_MIGRATIONS` and `SUPPRESS_SEED_CALLS` to truthy values.

I'll take this out.